### DISCUSSION TITLE
How ClusterScanType supposed to be used

### DISCUSSION THREAD

**Discussion by paraddise** [secureCodeBox/secureCodeBox#2701] [General]
I there is CRD `ClusterScanType`, I managed to create it, same as usual `ScanType`. 
But do scanners support it? maybe controller? 
Or it just generated by kubebuilder for future use?
Where container supposed to be created?

**Comment by J12934** [comment]
Hi

The Cluster Wide Resources are still considered to be somewhat experimental.
You can find more abou their history here: https://www.securecodebox.io/docs/architecture/architecture_decisions/adr_0012

In general they are supposed to work exacltly like the "normal" namespaced scantypes but allow you to define them across the cluster for scans in all namespaces to use. Scans are still namespaced, so the pods will be created in the namespace of the scan.

This was in part designed to make the auto discovery easier to use by allowing you to thefine these ScanTypes, ParseDefinitions and ScanCompletionHooks once per cluster.
